pipeline/deploy.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set +x
+
+if [[ -z "${VSPHERE_CSI_CONTROLLER_IMAGE}" ]]
+then
+	echo "Env variable unset: VSPHERE_CSI_CONTROLLER_IMAGE"
+	exit 1
+fi
+
+if [[ -z "${VSPHERE_SYNCER_IMAGE}" ]]
+then
+	echo "Env variable unset: VSPHERE_SYNCER_IMAGE"
+	exit 1
+fi
+
+# Borrow a testbed from CSI Testbed Pool Svc.
+testbed=`curl -X 'GET' ${CNS_TESTBEDPOOL_SVC_URL}`
+if [ $? -ne 0 ]
+then
+	echo "Unable to borrow a testbed!"
+	exit 1
+fi
+echo "TestbedInfo: $testbed"
+
+vcIp=`echo $testbed | jq '.vcIp'|tr -d '"'`
+if [ $? -ne 0 ]
+then
+	echo "Error getting the vcIp!"
+	exit 1
+fi
+
+echo "VC_IP=$vcIp" >> build.env
+
+svAdminCreds=`echo $testbed | jq '.svAdminCreds'|tr -d '"'|base64 -d`
+if [ $? -ne 0 ]
+then
+	echo "Error getting the svAdminCreds!"
+	exit 1
+fi
+echo "svAdminCreds = $svAdminCreds"
```

Comment:
nit: Ideally this should be named as svKubeConfig everywhere. There's no such thing called 'admin creds'.

Reply:
This is coming from the current testbed pool svc API, so keeping the naming consistent.